app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+def copy_file(command: str) -> None | str:
+    if len(command.split()) != 3:
+        return "Incorrect command is printed"
+    cmd, source_path, destination_path = command.split()
+    if cmd != "cp":
+        return "Command is not exist"
+    if source_path == destination_path:
+        return "The name of copied file is the same as original name"
+    with open(source_path, "r") as original, open(
+        destination_path, "w"
+    ) as copied:

```

Comment:
original and copied are not very descriptive variable names. Use `source_file` and `target_file` for example